admin/aioseop_module_class.php

```diff
@@ -2245,10 +2245,14 @@ function do_multi_input( $args ) {
 		}
 
 		/**
+		 * Get Option HTML
+		 *
 		 * Outputs a setting item for settings pages and metaboxes.
 		 *
-		 * @param $args
+		 * @since ?
+		 * @since 2.12 Add 'input' to allowed tags with 'html'. #2157
 		 *
+		 * @param $args
```

Comment:
add the description of the parameter as per WPCS

Reply:
It also needs the Type added, as well as @type for the description. Which I'd prefer breakpointing multiple instances since it seemed the values were dynamic, and so I held off on it, but yea definitely!